lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,315 @@
+                      ),
+                    ],
+                  ),
+                ),
+                Container(
+                  decoration: BoxDecoration(
+                    border: Border(
+                      bottom: new BorderSide(color: Colors.black12),
+                    ),
+                  ),
+                  child: Column(
+                    children: [
+                      ListTile(
+                        title: Text('Vocalization',
+                            style: CustomTypography.Typography.title()),
+                        subtitle: Text(widget.tile.vocalization,
+                            style: CustomTypography.Typography.subTitle()),
+                        trailing: Wrap(
+                          children: [
+                            Icon(Icons.keyboard_arrow_right,
+                                size: 35.0, color: Colors.black54),
+                          ],
+                        ),
+                        onTap: () => {
+                          Navigator.push(
+                              context,
+                              MaterialPageRoute(
+                                  builder: (context) => TileVocalization(
+                                        tileVocalization:
+                                            widget.tile.vocalization,
+                                      )))
+                        }, // Debug: change to edit
+                      ),
+                    ],
+                  ),
+                ),
+                Container(
+                  decoration: BoxDecoration(
+                    border: Border(
+                      bottom: new BorderSide(color: Colors.black12),
+                    ),
+                  ),
+                  child: Column(
+                    children: [
+                      ListTile(
+                        title: Text('Title Background Color',
+                            style: CustomTypography.Typography.title()),
+                        subtitle: Padding(
+                          padding: EdgeInsets.only(top: 10),
+                          child: Wrap(
+                            children: [
+                              for (var colorCircular in widget.editColors)
+                                InkResponse(
+                                  child: new Container(
+                                    margin: const EdgeInsets.only(right: 26.8),
+                                    width: screenSize.width / 18,
+                                    height: screenSize.width / 18,
+                                    decoration: new BoxDecoration(
+                                        color: colorCircular,
+                                        shape: BoxShape.circle),
+                                  ),
+                                ),
+                              // SizedBox(
+                              //   width: 30,
+                              // )
```

Comment:
@tim-nguyen-cs  I try to create a loop in Wrap to generate repetitive widgets from a List, but I can't put more than 1 widget in the loop (here I have `InkResponse` and `SizedBox`).  Do you have any suggestions on how I can include the SizedBox into the loop as well? (I currently use margin in the container in the replace, but it doesn't really match the design